docker/deepnote/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM wrangelvid/drake:gcs-science-robotics
+
+RUN apt-get -q update && apt-get -q install -y --no-install-recommends git nginx-light xvfb && apt-get -q clean
+ENV DISPLAY=:1
+ENV MOSEKLM_LICENSE_FILE="/tmp/mosek.lic"
+RUN git clone https://github.com/RobotLocomotion/gcs-science-robotics.git
+RUN pip3 install lxml matplotlib networkx numpy pandas pydot scipy
```

Comment:
why do we need these for deepnote, but not for the docker more generally?

Reply:
Putting the docker files should resolve this question.